site/src/comparison.rs

```diff
@@ -158,9 +158,20 @@ async fn populate_report(
         (None, None) => return,
     };
 
-    let entry = report.entry(direction).or_default();
+    let include_in_triage = match (primary.largest_change(), secondary.largest_change()) {
+        (Some(c), _) if c.magnitude() >= Magnitude::Medium => true,
+        (_, Some(c)) if c.magnitude() >= Magnitude::Medium => true,
+        _ => {
+            let primary_n = primary.num_changes();
+            let secondary_n = secondary.num_changes();
+            (primary_n * 2 + secondary_n) >= 6
```

Comment:
Please add a comment here explaining that this is somewhat arbitrary, but seems reasonable.

Reply:
👍I'll do this in a follow up PR since I want to also do it for the logic for perf bot comments.